### ISSUE TITLE
Load & Display Student Performance Stats 

### ISSUE THREAD

**Issue by Accuracy-exe** [acmpesuecc/Revisor#3]
The Performance Tab is for displaying the average score of all the tests the student has taken as a percentage value, the number of tests taken and a graph to show variations in performance of the student among the tests.
- Read history (list of test scores as percentage) from client_storage
- Plot graph based on history

This can be properly tested once The Test Conduction view is

**Comment by Accuracy-exe** [comment]
!assign @SavvyHex

**Comment by svaidy-i** [comment]
May I please be assigned to this issue

**Comment by Accuracy-exe** [comment]
Hey, 
This issue has been assigned to someone already

**Comment by Accuracy-exe** [comment]
!deassign @SavvyHex

**Comment by Accuracy-exe** [comment]
@SavvyHex Could you report your progress?

**Comment by SavvyHex** [comment]
I couldn't run the app, but I've started work on the Performance page using the documentation.

**Comment by Ferrari2025** [comment]
may i be assigned to this issue?

**Comment by MKD2004** [comment]
Can i get it assigned?

**Comment by QWERTY-Ak** [comment]
can i please be. assigned this

**Comment by Accuracy-exe** [comment]
> I couldn't run the app, but I've started work on the Performance page using the documentation.

@SavvyHex Do you wish to continue to work on this? And as for making sure flet works, you'll have to set up flutter as well
I'll update the README for this
If u wish to continue please ping me asap

**Comment by SavvyHex** [comment]
Yes, and could you please help me set this up with flutter? I'm in the 3rd row from the front

**Comment by Accuracy-exe** [comment]
Alright, Room Number?

**Comment by SavvyHex** [comment]
Seminar hall 1

**Comment by SavvyHex** [comment]
@Accuracy-exe My bad, I'm in sem hall 2, the one on the ground floor TT

**Comment by Accuracy-exe** [comment]
@SavvyHex I did come to Sem Hall 1 but u don't seem to be there, u can just google up download flutter that should guide you

**Comment by SavvyHex** [comment]
Ahh I've been assigned another issue right now...
Please go ahead and assign it to someone else

**Comment by aryanmittal2107** [comment]
can yo assign me this issue

**Comment by Accuracy-exe** [comment]
!assign @aryanmittal2107

**Comment by aryanmittal2107** [comment]
!extend 30

**Comment by Accuracy-exe** [comment]
!extend 30

**Comment by aryanmittal2107** [comment]
i am having  problems in my flet

**Comment by Accuracy-exe** [comment]
> i am having problems in my flet
@aryanmittal2107 
Make sure you have correctly installed flutter (it is a dependency)

**Comment by aryanmittal2107** [comment]
its installed properly

**Comment by aryanmittal2107** [comment]
i have opened revisor on flet

**Comment by aryanmittal2107** [comment]
!withdraw

**Comment by likhita-marella** [comment]
hey, can i get assigned to this issue? thanks.

**Comment by Accuracy-exe** [comment]
!assign @likhita-marella 50

**Comment by Accuracy-exe** [comment]
You are already assigned to another issue @likhita-marella

**Comment by likhita-marella** [comment]
no i withdrew from that

**Comment by Accuracy-exe** [comment]
!assign @likhita-marella 50

**Comment by codebuddyjr** [comment]
assign me this pls

**Comment by Sami9692** [comment]
can i get assigned to thisss??

**Comment by Accuracy-exe** [comment]
!assign @Sami9692

**Comment by Sami9692** [comment]
withdraw

**Comment by Accuracy-exe** [comment]
!deassign

**Comment by sampada115** [comment]
can you assign me this?

**Comment by sampada115** [comment]
plss

**Comment by sampada115** [comment]
@Accuracy-exe

**Comment by Aditya-C-S** [comment]
can u please assign it to me @Accuracy-exe

**Comment by devanshishukla17** [comment]
can I get this assigned please

**Comment by Ruchavw** [comment]
can I get assigned to this please?

**Comment by AutisticCoder-9000** [comment]
Can I get this issue assigned to me?

**Comment by Hemz1416** [comment]
can i get assigned to this issue

**Comment by Alex-Hunterz** [comment]
Can i get assigned to this issue?
